ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/InsertsStreamEndpoint.java

```diff
@@ -89,7 +86,8 @@ private DataSource getDataSource(
     final DataSource dataSource = metaStore.getSource(sourceName);
     if (dataSource == null) {
       throw new KsqlApiException(
-          "Cannot insert values into an unknown stream: " + sourceName, ERROR_CODE_BAD_STATEMENT);
+          "Cannot insert values into an unknown stream/table: " + sourceName,
+            ERROR_CODE_BAD_STATEMENT);
     }
 
     if (dataSource.getKsqlTopic().getKeyFormat().isWindowed()) {
```

Comment:
Can we also update the error message in this block to say `stream/table` rather than just `stream`?